charts/galoy/charts/lnpay/templates/deployment.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "lnpay.fullname" . }}
   labels:
     {{- include "lnpay.labels" . | nindent 4 }}
```

Comment:
do we still need this `lnpay.labels`?

Reply:
or maybe, aslk differently, what is the difference between the include here and the the include below?